not-so-trivial-pursuit/client/src/components/question-render.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from "react";
+
+const RadioButton = ({answer, index, onValueChange}) => {
+ let id = `${index}-choice`;
+  return( 
+  <div>
+    <input type="radio" id={id} name="answer" value={answer} onChange={onValueChange}/>
+    <label for={id}>{answer}</label>
+  </div>  
+)};
+
+const QuestionRender = ({question}) => {
+  const[isRight, setIsRight] = useState(false);
+  const handleSubmit = (evt) => {
+    evt.preventDefault();
+    console.log(evt.target);
+}
+  const onValueChange = (evt) => {
+    console.log(evt.target.value);
+    console.log(evt.target.value===question.correct_answer);
+    if(evt.target.value===question.correct_answer){
```

Comment:
if you do this in handleSubmit, you can set whether or not the user is right after they've pressed "submit", instead of any time that they click a radio button!

Reply:
I tried to move it there, but then it broke so i moved it back and gave up for now. lol